pkg/scheduler/scheduler.go

```diff
@@ -111,7 +117,7 @@ func (s *Scheduler) schedule(ctx context.Context) {
 			continue
 		}
 		if s.queues.RequeueWorkload(ctx, &w) {
-			log.V(2).Info("Workload requeued", "workload", klog.KObj(w.Obj), "queue", klog.KRef(w.Obj.Namespace, w.Obj.Spec.QueueName))
+			log.V(2).Info("Workload requeued", "queuedWorkload", klog.KObj(w.Obj), "queue", klog.KRef(w.Obj.Namespace, w.Obj.Spec.QueueName))
```

Comment:
is log inited with `queuedWorkload` in L101?

Reply:
No, because that log is overwritten inside the loop (`:=`)

Reply:
Sorry, I'm missing some code.